app/views/admin/shared/_search_bar.slim

```diff
@@ -0,0 +1,3 @@
+= form_tag url, method: :get do
```

Comment:
@denysmedynskyi
I'm not sure about that, this piece of code is solely for demonstration purposes due to the fact that there's no design & no specific requirements regarding search functionality for different models.

It would be cool if you could create a separate ticket with this information provided, sounds good?